library/Services/Librato/Annotations.php

```
@@ -0,0 +1,148 @@
+<?php
+namespace Services\Librato;
+
+use \Services\Librato;
+
+use \HTTP_Request2 as Request2;
+use \HTTP_Request2_Response as HttpResponse;
+use \HTTP_Request2_Exception as HttpException;
+
+/**
+ * @category   Services
+ * @package    Services_Librato
+ * @subpackage Services_Librato_Annotations
+ * @author     Ulf Härnhammar <ulfharn@gmail.com>
+ * @author     Till Klampaeckel <till@lagged.biz>
+ * @version
+ * @license
+ * @link
+ */
+class Annotations extends Librato
+{
+    /**
+     * @var string $apiKey
+     */
+    protected $apiKey;
+
+    /**
+     * @var string $endpoint
+     */
+    protected $endpoint = 'https://metrics-api.librato.com/v1';
+
+    /**
+     * @var string $user Most likely the email address of your account.
+     */
+    protected $user;
+
+    /**
+     * __construct
+     *
+     * @param string $user   the user
+     * @param string $apiKey the API key
+     *
+     * @return $this
+     */
+    public function __construct($user, $apiKey)
```

I am wondering if we should throw `\InvalidArgumentException` when either is not set. What do you think?